static/main.css

```diff
@@ -232,4 +232,70 @@ main.container {
 .create-event-button:hover {
   background-color: #D85536;
   border: solid 2px #D85536;
+}
+
+.white-background-button {
+  width: auto;
+  height: auto;
+  padding: 9px 20px 7px 19px;
+  border-radius: 24px;
+  border: solid 2px #e84a27;
+  background-color: #fff;
+  color:#002855;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.white-background-button:hover {
```

Comment:
The white-background-button:hover is same as white-background-button. The color, background-color and border attributes are the same.

Reply:
This is intended to override the Bootstrap hover color.

Reply:
that makes sense. Thanks for clarifying that. I was able to test it out as well.

Maybe you could combine the button main css and the hover css styles like below :  
```
.create-event-button, .create-event-button:hover, .create-event-button:active, .create-event-button:link {
  width: 140px;
  margin-left:20px;
  background-color: #D85536;
  border-radius: 24px;
  border: solid 2px #D85536;
  font-family: proxima-nova, sans-serif;
}
```

Reply:
Sure, I think that is cleaner.